utils/hparams.py

```diff
@@ -11,10 +11,11 @@ class Hparams:
 
     # Preprocessed files
     default_dir = 'dataset/Flickr8k/prepro'
-    parser.add_argument('--train', default='%s./train_set.pkl' % default_dir)
-    parser.add_argument('--dev', default='%s./dev_set.pkl' % default_dir)
-    parser.add_argument('--test', default='%s./train_set.pkl' % default_dir)
-    parser.add_argument('--vec2word', default='%s./vec2word_model.npy' % default_dir)
+    parser.add_argument('--train', default='%s/train_set.pkl' % default_dir)
+    parser.add_argument('--dev', default='%s/dev_set.pkl' % default_dir)
+    parser.add_argument('--minidev', default='%s/minidev_set.pkl' % default_dir)

```

Comment:
I dont think that adding a new parameter is necessary.
The same can be achieved by using either the train or dev split, with setting --split_size to a small value. So split_size=0.1 and dev of 1000 objects gives a dataset of 100 objects